x/auth/client/cli/tx_sign.go

```diff
@@ -60,11 +60,11 @@ account key. It implies --signature-only.
 }
 
 func makeSignBatchCmd() func(cmd *cobra.Command, args []string) error {
```

Comment:
This function is wayyyy to large for naked returns. If using named return value(s), please avoid naked returns and explicitly return the value(s).

Reply:
ah, I merged the PR into #8121 already. Could you put the same comment on #8121?

Reply:
Yeah you're right. And the naked returns + closure are actually required only after line 95 https://github.com/cosmos/cosmos-sdk/pull/8122/files#diff-eb0834c140c5c40cca7f39ab9d334536d3529642fce425754b8608c5e79b6430R95